src/test/java/gov/usgs/cida/pubs/webservice/pw/PwPublicationMvcServiceTest.java

```diff
@@ -80,13 +77,14 @@ public void setup() {
 			xmlBusService,
 			configurationService,
 			templateConfiguration,
-			publicationBusService
+			contentStrategy
 		);
 
-		filters = new HashMap<>();
-		filters.put(BaseDao.PAGE_SIZE, "13");
-		filters.put(BaseDao.PAGE_ROW_START, "4");
-		filters.put(BaseDao.PAGE_NUMBER, "8");
+		filters = new PwPublicationFilterParams();
+		filters.setPage_size("13");
+		filters.setPage_row_start("4");
+		filters.setPage_number("8");
```

Comment:
These setters are using a mixture of cases, do we want to consolidate them to something like `setPageSize, setPageRowStart, setPageNumber`?

Reply:
They need to be cased like this in order for Spring to set them from the query parameters of the (html) query.